manifest.toml

```
@@ -0,0 +1,70 @@
+packaging_format = 2
+
+id = "discourse"
+name = "Discourse"
+description.en = "Discussion platform"
+description.fr = "Plateforme de discussion"
+
+version = "2.8.14~ynh1"
+
+maintainers = ["JimboJoe"]
+
+[upstream]
+license = "GPL-2.0"
+website = "http://Discourse.org"
+demo = "https://try.discourse.org"
+code = "https://github.com/discourse/discourse"
+
+[integration]
+yunohost = ">= 11.1.20"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "1G"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/forum"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.admin]
+    type = "user"
+
+[resources]
+    [resources.sources]
+        [resources.sources.ldap-auth]
+        url = "https://github.com/jonmbake/discourse-ldap-auth/archive/v0.6.0.tar.gz"
+        sha256 = "1f64f90f648f53b40608912221adb60d86c8c13856aaba68c645cd88279445d4"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.main]
+        url = "https://github.com/discourse/discourse/archive/v2.8.14.tar.gz"
+        sha256 = "b4b9a2857515ab79b2bc71cc0bdd09f9bf749d506c095b76d86846c9adf62623"
+        autoupdate.strategy = "latest_github_tag"
+
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "g++ 'libjemalloc1|libjemalloc2' libjemalloc-dev zlib1g-dev libreadline-dev libpq-dev libssl-dev libyaml-dev libcurl4-dev libapr1-dev libxslt1-dev libxml2-dev vim imagemagick postgresql postgresql-server-dev-all postgresql-contrib optipng jhead jpegoptim gifsicle brotli postgresql postgresql-client imagemagick libjemalloc1|libjemalloc2 libcurl4-openssl-dev libyaml-dev libxml2-dev libpq-dev libreadline-dev brotli libunwind-dev libtcmalloc-minimal4 cmake pngcrush pngquant advancecomp jhead jpegoptim libjpeg-turbo-progs optipng"
```

Tests are currently failing because of the `|` ... I think that's for backward compat with Buster, of maybe forward compat with Bookworm ?

Anyway there doesnt seem to be any `libjemalloc1` in either Buster or Bullseye

Soo lets simplify it ? 

```suggestion
    packages = "g++ libjemalloc2 libjemalloc-dev zlib1g-dev libreadline-dev libpq-dev libssl-dev libyaml-dev libcurl4-dev libapr1-dev libxslt1-dev libxml2-dev vim imagemagick postgresql postgresql-server-dev-all postgresql-contrib optipng jhead jpegoptim gifsicle brotli postgresql postgresql-client imagemagick libjemalloc1|libjemalloc2 libcurl4-openssl-dev libyaml-dev libxml2-dev libpq-dev libreadline-dev brotli libunwind-dev libtcmalloc-minimal4 cmake pngcrush pngquant advancecomp jhead jpegoptim libjpeg-turbo-progs optipng"
```